service-api/app/features/context/Acceptance/LpaContext.php

```diff
@@ -102,7 +102,7 @@ public function iHaveBeenGivenAccessToUseAnLPAViaAPaperDocument(): void
 
         $this->userPostCode          = 'string';
         $this->userFirstnames        = 'Ian Deputy';
-        $this->userSurname           = 'Deputy';
+        $this->userSurname           = 'D’eputy';
```

Comment:
```suggestion
        $this->userSurname           = 'Deputy';
```
In terms of testing for this I think it might better to have either a new scenario or just rely on the unit tests.